docs/source/_json/collection.json

```diff
@@ -5,8 +5,19 @@ HTTP 200 OK
 content-type: application/json
 
 {
+  "@components": {
+    "breadcrumbs": {
+      "@id": "http://localhost:55001/plone/collection/@components/breadcrumbs"
+    }, 
+    "navigation": {
+      "@id": "http://localhost:55001/plone/collection/@components/navigation"
+    }
+  }, 
   "@id": "http://localhost:55001/plone/collection", 
   "@type": "Collection", 
+  "@workflow": {
```

Comment:
@tisto Right now I'm using the "@"-prefix mainly for namespacing reasons. But that problem is not fully solved yet. We need a solution for handling name conflicts with field names if fields are on top level of the JSON representation.